resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
+    name = 'Reverse complement FASTQ'
```

Comment:
Name could include that this is a single-end process. My suggestion would be to use `(single-end)`, as do other processes in resolwe-bio.
![Screenshot from 2019-11-04 21-05-38](https://user-images.githubusercontent.com/544571/68154013-eb2e8980-ff46-11e9-8f89-39f871938f1e.png)